src/TractionRecClient.php

```diff
@@ -138,11 +138,18 @@ public function getAccessToken(): string {
     }
     catch (RequestException $e) {
       $response = $e->getResponse();
+      $this->logger->error($e->getMessage());
     }
 
     $token_request_body = $response->getBody()->getContents();
 
     $access_token = json_decode($token_request_body);
+    // Return empty string if no token can be retrieved.
+    // Error will be logged elsewhere.
```

Comment:
let's log error to Drupal log

Reply:
@podarok the error is logged on line 141. I'd added a log statement here and removed it because it was just duplicating the one above.